src/test/java/com/wikia/webdriver/testcases/adstests/TestUap.java

```diff
@@ -33,6 +35,27 @@ public void adsUapOasis(Page page,
     verifySlotsUnblocked(ads, ListUtils.union(atfSlots, btfSlots));
   }
 
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "adsUapMercury",
+      groups = "AdsUapMercury"
+  )
+  public void adsUapMercury(Page page,
+                            List<Map<String, Object>> atfSlots,
+                            List<Map<String, Object>> btfSlots) {
+    AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page));
+    verifySlotsUnblocked(ads, atfSlots);
+    verifySlotsBlocked(ads, btfSlots);
+    ads.scrollToSlot(AdsContent.SLOTS_SELECTORS.get("MOBILE_IN_CONTENT_CONTAINER"));
```

Comment:
`s/"MOBILE_IN_CONTENT_CONTAINER"/AdsContent.MOBILE_IN_CONTENT_CONTAINER`

Actually we need to use `getSlotSelector` and make SLOTS_SELECTORS to be private.